### DISCUSSION TITLE
WT32-ETH01 no ethernet connection

### DISCUSSION THREAD

**Discussion by Monsterchef** [arendst/Tasmota#18115] [Q&A] [answered]
Hey erveryone...
Can someone tell me the settings for the WT32-ETH01?
My Problem is that the flashing with the Standard "ESP32" settings are no Problem.
But after the WLan Configuration, setting the right Template an enable Ethernet over Console (EthAddress 1) .
I cant bring him into the "normal" Ethernet, Wifi works fine!

Anyone has a idea what i have doing wrong?

best regards
![grafik](https://user-images.githubusercontent.com/79506831/222951285-2aff995c-77bc-46c4-8b7b-adca1dcdd6ae.png)
![grafik](https://user-images.githubusercontent.com/79506831/222951318-9833e4f3-caa5-4893-9dcb-a23bb463064e.png)
![grafik](https://user-images.githubusercontent.com/79506831/222951376-8286bfec-c918-45ca-aed0-a72ec4b9a7bf.png)

**Comment by sfromis** [comment, marked as answer]
What version is your board? I've got my V1.2 board (printed on the back of the PCB) working with 
https://templates.blakadder.com/wireless_tag_WT32-ETH01

```
13:05:33.127 CMD: Backlog Ethernet;EthAddress;EthClockMode;EthType
13:05:33.159 MQT: xxxxxxxx/stat/ETHERNET = {"Ethernet":"ON"}
13:05:33.382 MQT: xxxxxxxx/stat/ETHADDRESS = {"EthAddress":1}
13:05:33.632 MQT: xxxxxxxx/stat/ETHCLOCKMODE = {"EthClockMode":0}
13:05:33.881 MQT: xxxxxxxx/stat/ETHTYPE = {"EthType":0}
```

To have Ethernet as default, but fallback to Wifi if unavailable, I can recommend something like the Berry code from:
https://tasmota.github.io/docs/Berry-Cookbook/#ethernet-network-flipper

**Reply by Monsterchef** [reply]
Hey, thanks for your Answer.
My board is also 1.2 marked on the back of the PCB.
Which exactly steps have you done after flashing?
Maybe I was just being stupid...

**Reply by sfromis** [reply]
Your log shows that you did not activate the template.

**Reply by Monsterchef** [reply]
Hey youre right...
I thought I activated it...
Interestingly, the module has already adapted, so I didn't notice it.
Thanks for help by this stupid problem!
